lookout/style/typos/datasets_preparation.py

```diff
@@ -108,3 +123,37 @@ def prepare_data(params: Optional[Mapping[str, Any]] = None) -> pandas.DataFrame
     prepared_data = filter_splits(flat_data, vocabulary_tokens)[[Columns.Frequency, Columns.Split,
                                                                  Columns.Token]]
     return prepared_data
+
+
+def tune_fasttext_model(data: pandas.DataFrame,
+                        params: Optional[Mapping[str, Any]] = None) -> None:
+    """
+    Tune fasttext model on given data.
```

Comment:
Is it a hyperparameter optimization? If it is, replace the function description, because currently it repeats the name and hence is quite useless.